src/renderer/screens/manager/FirmwareUpdate/index.js

```diff
@@ -103,68 +105,41 @@ class FirmwareUpdate extends PureComponent<Props, State> {
 
     if (!firmware) {
       if (!isDeprecated) return null;
-
       return (
-        <Box
-          py={2}
-          px={4}
-          bg="blueTransparentBackground"
-          horizontal
-          alignItems="center"
-          justifyContent="space-between"
-          borderRadius={1}
-        >
-          <Box flex="1" mr={4}>
-            <Text ff="Inter|SemiBold" fontSize={4} color="palette.primary.main">
-              <Trans i18nKey="manager.firmware.deprecated" />
-            </Text>
-          </Box>
-          <Button
-            primary
-            onClick={() => {
-              openURL(urls.contactSupport);
-            }}
-          >
-            <Trans i18nKey="manager.firmware.contactSupport" />
-          </Button>
-        </Box>
+        <FirmwareUpdateBanner
+          old
+          right={
+            <FakeLink onClick={() => openURL(urls.contactSupport)}>
+              <Trans i18nKey="manager.firmware.banner.old.cta" />
+            </FakeLink>
+          }
+        />
       );
     }
 
     return (
-      <Box
-        py={2}
-        px={4}
-        bg="blueTransparentBackground"
-        horizontal
-        alignItems="center"
-        justifyContent="space-between"
-        borderRadius={1}
-      >
-        <Box flex="1">
-          <Text ff="Inter|SemiBold" fontSize={5} color="palette.primary.main">
-            <Trans
-              i18nKey="manager.firmware.latest"
-              values={{ version: getCleanVersion(firmware.final.name) }}
-            />
-          </Text>
-        </Box>
-
-        {manager.firmwareUpdateRequiresUserToUninstallApps(device.modelId, deviceInfo) && (
-          <Box px={4} horizontal alignItems="center" color="palette.primary.main">
-            <IconInfoCircle size={12} />
-            <Text style={{ marginLeft: 6 }} ff="Inter" fontSize={4}>
-              <Trans i18nKey="manager.firmware.removeApps" />
-            </Text>
-          </Box>
-        )}
-        <UpdateFirmwareButton
-          deviceInfo={deviceInfo}
-          firmware={firmware}
-          onClick={this.handleDisclaimerModal}
-          disabled={disableFirmwareUpdate}
+      <>
+        <FirmwareUpdateBanner
+          right={
+            <Box alignItems={"flex-end"} horizontal>
```

Comment:
Good to know, curly braces are no longer required when dealing with string props. From my mind they add some noise.

Reply:
Old habits die hard, I suggest you raise the concern to the core team and ask them to add something like https://github.com/yannickcr/eslint-plugin-react/blob/HEAD/docs/rules/jsx-curly-brace-presence.md to our eslint so that it's handled automatically